shared/src/commonMain/kotlin/io/customer/shared/util/DateTimeUtil.kt

```diff
@@ -0,0 +1,23 @@
+package io.customer.shared.util
+
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+
+internal interface DateTimeUtil {
+    val now: Instant
+    val nowUnixTimestamp: Long
+
+    fun toUnixTimestamp(time: Instant): Long
+}
```

Comment:
Tried to keep this and other util classes much similar to what we have already in our SDK.